tools/provenance-generator/pkg/testdata/prowjob.yaml

```diff
@@ -111,15 +111,3 @@ spec:
       report: true
   prowjob_defaults:
     tenant_id: GlobalDefaultID
-status:
```

Comment:
why is this being removed both in json and yaml?

Reply:
The yaml is loaded as a fakeclient to test fetching ProwJobs. I have decided not to include the status field as it is not complete.